File: CSharp/PaymentsService/Infrastructure/Payments/CreditCard/ProcessCreditCardPaymentResponse.cs
using PaymentsService.Domain.Payments;

namespace PaymentsService.Infrastructure.Payments.CreditCard;
public sealed class ProcessCreditCardPaymentResponse : ProcessPaymentBaseResponse
{
    public ProcessCreditCardPaymentResponse() 
        : base("CreditCard")
    {
    }

    public string TransactionId { get; set; }
    public string ReceiptId { get; set; }
}

File: Python/PaymentsService/Infrastructure/Payments/CreditCard/ProcessCreditCardPaymentResponse.py
from Domain.Payments.ProcessPaymentBaseResponse import ProcessPaymentBaseResponse


class ProcessCreditCardPaymentResponse(ProcessPaymentBaseResponse):
    def __init__(self, transactionId = None, receiptId = None, status = None, error = None):
        super().__init__("CreditCard", status, error)
        self.transactionId = transactionId
        self.receiptId = receiptId

    def to_dict(self):
        return {
            "paymentMethod": self.paymentMethod,
            "status": self.status,
            "error": self.error,
            "transactionId": self.transactionId,
            "receiptId": self.receiptId
        }